components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -32,14 +33,20 @@ use zerovec::ZeroMap;
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[yoke(prove_covariance_manually)]
 pub struct DateSymbolsV1<'data> {
+    /// Symbol data for months.
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub months: months::ContextsV1<'data>,
+    /// Symbol data for weekdays.
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub weekdays: weekdays::ContextsV1<'data>,
+    /// Symbol data for eras.
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub eras: Eras<'data>,
 }
 
+/// Symbol data for the day periods needed to format a time.
+/// 
+/// For more information on date time symbols, see [`FieldSymbol`](crate::fields::FieldSymbol).
```

Comment:
Nit: These docs are not quite accurate, and they are not required since this is a pub-crate type. Perhaps you attached them to the wrong struct?